specs/signing-and-verification-workflow.md

```diff
@@ -22,8 +22,8 @@ The user wants to sign an OCI artifact and push the signature to a repository.
             1. Verify that the timestamp signing algorithm satisfies [algorithm requirements](./signature-specification.md#signature-algorithm-requirements).
         1. Embed timestamp to the signature envelope.
 1. **Push the signature envelope:** Push the signature envelope generated in the previous step to the repository.
-1. **Generate signature artifact manifest:** As decribed in [signature specification](./signature-specification.md#storage) create the Notary v2 signature artifact manifest for the signature envelope generated in step 1.
-1. **Push signature artifact manifest:** Push Notary v2 signature artifact manifest to the repository.
+1. **Generate signature artifact manifest:** As described in [signature specification](./signature-specification.md#storage) create the Notary signature manifest for the signature envelope generated in step 1.
+1. **Push signature artifact manifest:** Push Notary signature manifest to the repository.
```

Comment:
**Push signature artifact manifest:** Push the Notary signature manifest to the repository.